.jenkins/actions/run_driver_parallel_regression_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e -x
+BACKEND=$1
+
+export TEST_ARGS="-v -s -rsx --backend=${BACKEND} "
+
+if [ ${python_env} == "virtualenv" ]; then
```

Comment:
Nice! I'm glad you were able to keep the same interface for running the tests even with the reference data folder structure changing.